### PR TITLE
Logs refining

### PR DESCRIPTION
**Pull request by vpodzime**

### CODE REVIEW

**Comment by vpodzime** [comment]
Replacement for #87

File: core/src/mender-client.c [hunk 1]
```diff
@@ -969,10 +969,10 @@ mender_client_update_work_function(void) {
...
                 }
 
-                mender_log_info("Downloading deployment artifact with id '%s', artifact name '%s' and uri '%s'",
-                                deployment->id,
-                                deployment->artifact_name,
-                                deployment->uri);
+                mender_log_debug("Downloading deployment artifact with id '%s', artifact name '%s' and uri '%s'",
+                                 deployment->id,
+                                 deployment->artifact_name,
+                                 deployment->uri);
```
Comment (lluiscampos) [inline core/src/mender-client.c:975]: I think this one should be info. Our PO has claimed several times that to have progress updates is a highly demanded feature.

**Review by lluiscampos — APPROVED** [review]

File: core/src/mender-client.c (hunk 1 above)
Comment (vpodzime) [reply core/src/mender-client.c:975]: But I guess the above is a nice first message. :+1:

Comment (lluiscampos) [reply core/src/mender-client.c]: Sorry, I got confused!

I didn't implicitly ask for restoring this one, but it makes sense as it gives context to the other ones (the progress)

**Pull request merged**